src/main/java/org/dita/dost/module/BranchFilterModule.java

```diff
@@ -376,7 +377,7 @@ private void generateCopies(final Element topicref, final List<FilterUtils> filt
                 final List<XMLFilter> pipe = singletonList(writer);
 
                 try {
-                    job.getStore().transform(srcAbsUri, dstAbsUri, pipe);
+                	job.getStore().transform(URLUtils.removeFragment(srcAbsUri), URLUtils.removeFragment(dstAbsUri), pipe);
```

Comment:
Fix indentation

Reply:
I removed this indentation, about this particular line it's not very efficient when you have multiple topicrefs to inner topics, like for example:

            <topicref class="- map/topicref " href="topic.dita#subtopic1_id"/>
            <topicref class="- map/topicref " href="topic.dita#subtopic2_id"/>
            <topicref class="- map/topicref " href="topic.dita#subtopic3_id"/>

we end up copying the "topic.dita" over "topic-1.dita" about 3 times although only once would have been enough.
Any ideas about how we could avoid this? So somehow if "dstAbsUri" is already on disk I would probably want to avoid copying content over it.

Reply:
@jelovirt - thanks for approving this pull request!

@raducoravu mentioned above that when there are inner topics, the fix will perform redundant copies, which is not very efficient. Can you think of a way this could be improved?